src/subscription_manager/gui/registergui.py

```diff
@@ -1287,11 +1341,13 @@ def _find_suitable_service_levels(self, consumer, facts):
                 len(self.backend.cs.partial_stacks) == 0:
             raise AllProductsCoveredException()
 
+        log.debug("presla")
```

Comment:
I dunno, whats up dog?  

Wait, I did that wrong. (presla == pre sla)

Cleaning up most of these debug statements now.